styles.css

```diff
@@ -542,6 +546,186 @@ body {
   transform: rotate(45deg);
 }
 
+.project-link1 {
+  border: 1px solid blue;
+  width: 120px;
+  height: 35px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  background-color: #fff;
+  color: #396df2;
+  font-size: 16px;
+}
+
```

Comment:
- [ OPTIONAL ] Kindly fix the position for the pop so you can make the width smaller and the height smaller so I can it in a clear way so please fix it 🙏🏻🙏🏻